codes.merritt.Nyrna.yml

```diff
@@ -0,0 +1,46 @@
+---
+app-id: codes.merritt.Nyrna
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: nyrna
+separate-locales: false
+finish-args:
+- --share=ipc
+- --socket=fallback-x11
+- --socket=wayland
```

Comment:
you just said it only support x11. so that's incorrect then.

Reply:
Thank you for pointing that out! <3